library/std/src/env.rs

```diff
@@ -491,7 +490,8 @@ pub struct JoinPathsError {
 /// }
 /// ```
 ///
-/// [`env::split_paths`]: split_paths
+/// [`env::join_paths()`]: join_paths
```

Comment:
Unused link.

Reply:
Do we have error on unused link?

Reply:
```suggestion
```

Reply:
> Do we have error on unused link?

No, but it's confusing for people editing the docs. See https://github.com/rust-lang/rust/issues/62345.